src/App.tsx

```diff
@@ -1,15 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(
+    ({ title, description }) => (
+      title.toLowerCase().includes(query.toLowerCase().trim()))
+    || description.toLowerCase().includes(query.toLowerCase().trim()),
```

Comment:
Don't repeat yourself. Create a method for this logic `.toLowerCase().includes(query.toLowerCase().trim()))`